src/pages/Legislation.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Layout from "../components/layout";
+import SEO from "../components/seo";
+
+const Legislation = () => (
+    <Layout>
+        <SEO title={"Legislation vocabulary"} />
+        <h1>Life Engine vocabulary - Legislation</h1>
+            <a name={"attribution"}><h2>Attribution</h2></a>
+            <p>Text required to give proper attribution to data source</p>
+            <a name={"name"}><h2>Name</h2></a>
+            <p>Legal name of a company or first name and last name of a person.</p>

```

Comment:
Is this defintion of a person's name really what's used in legal context; isn't usually the full name used; all first names and the last name? Just thinking aloud, I'm definitely not an expert at all in this field.

Reply:
This is for now defined like this, and it has been approved.